api/src/BcGov.Malt.Web/Models/SharePoint/GetSiteGroupsVerboseResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace BcGov.Malt.Web.Models.SharePoint
+{
+    public class GetSiteGroupsVerboseResponse : ODataVerboseResponse<ODataResultsCollection<SiteGroup>>
+    {
+    }
+}
```

Comment:
nit: a few of these files are missing newline at end of file, and github is pointing it out 😄 not important but also easy to update :)